tests/test_smoke.py

```diff
@@ -0,0 +1,88 @@
+import yaml
+import subprocess
+
+
+def test_upload():
+    with open("rockcraft.yaml") as file:
+        rockcraft = yaml.safe_load(file)
+        name = rockcraft["name"]
+        version = rockcraft["version"]
+
+        subprocess.run(
+            [
+                "skopeo",
+                "copy",
+                f"oci-archive:{name}_{version}_amd64.rock",
+                f"docker-daemon:{name}:test",
+            ]
+        )
+
+
+def test_all_apps():
+    with open("rockcraft.yaml") as file:
+        rockcraft = yaml.safe_load(file)
+        name = rockcraft["name"]
+        version = rockcraft["version"]
+        major_version = version.split(".")[0]
+
+        override = {
+            "pgbackrest": "version",
+        }
+
+        apps = [
```

Comment:
There's no manifest of apps and services like in the snap, so we have to maintain a manual list. Copied what the snap is checking ATM.